tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -423,6 +434,167 @@ RHODS Notification Drawer Should Contain
     Click Element    xpath=//*[contains(@class,'notification-badge')]
     Page Should Contain Element
     ...    xpath=//*[contains(text(),'${message}')]
+    Close Notification Drawer
+
+Open Notebook Images Page
+    [Documentation]    Opens the RHODS dashboard and navigates to the Notebook Images page
+    Page Should Contain    Settings
+    Menu.Navigate To Page    Settings    Notebook Images
+    Wait Until Page Contains    Notebook image settings
+    Page Should Contain    Notebook image settings
+
+Import New Image
+    [Documentation]  software and packages should be lists of lists
+    [Arguments]  ${repo}  ${name}  ${description}  ${software}  ${packages}
+    Sleep  1
```

Comment:
Is this Sleep needed here? Could it be replaced by Wait Until ... ?

Reply:
Yes, it's needed. For some reason in the screenshots the button I need is already shown but the test would sometimes fail because it couldn't find it.